oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/AuthorizedClientServiceReactiveOAuth2AuthorizedClientManager.java

```diff
@@ -134,4 +263,106 @@ public void setContextAttributesMapper(Function<OAuth2AuthorizeRequest, Mono<Map
 			return Mono.fromCallable(() -> mapper.apply(authorizeRequest));
 		}
 	}
+
+	/**
+	 * The default authorization success handler,
+	 * which will save the authorized client in a
+	 * {@link ReactiveOAuth2AuthorizedClientService}.
+	 *
+	 * @since 5.3
+	 */
+	public static class DefaultAuthorizationSuccessHandler implements ReactiveOAuth2AuthorizationSuccessHandler {
```

Comment:
I'd rather make this `private` or implement via a `private` helper method. The implementation is quite minimal so code reuse wouldn't make much of a difference here. I'm ok with minimal code duplication vs. exposing a public API. Easier from a maintenance standpoint on our end and allows us to change things more easily.

Reply:
Pulled out into public outer class as per other discussion in ServerOAuth2AuthorizedClientExchangeFilterFunction